docs/source/quickstart.rst

```diff
@@ -241,6 +241,12 @@ where ``[60]`` can be replaced with a list of the visit numbers you want to
 download, e.g. ``[60, 61, 62, 63, 64]`` if you want the entire transit
 observation.
 
+.. warning::
+	HST planning is usually consistent between orbits, but may sometimes modify 
+	the number of exposures in a scan for observation timing purposes. Ensure 
+	that your datasets have consistent numbers of exposures, or exclude 
+	trailing exposures at the end of orbits if needed.
+
```

Comment:
I'm not sure this is the best place for this warning. Thoughts @kevin218?

Reply:
I thought the issue was with a change in the number of up-the-ramp samples (i.e. non-destructive reads).

Reply:
Sorry, yes, it was non-destructive reads.

Reply:
Ok, great. 

Now going back to Taylor's concern, we could put it in the faq.

Reply:
Sounds good. I've moved that warning to the FAQ, and made sure the appropriate errors get thrown to inform the user of the issue before they try to go further in the analysis and get stuck.